server/src/main/java/com/config/RestTemplateConfig.java

```diff
@@ -0,0 +1,32 @@
+package com.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.client.ClientHttpRequestFactory;
+import org.springframework.http.client.SimpleClientHttpRequestFactory;
+import org.springframework.http.converter.StringHttpMessageConverter;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.client.RestTemplate;
+
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+
+@Configuration
+public class RestTemplateConfig {
```

Comment:
`RestTemplateConfig` is a pre-defined restful style http client, I think. I guess that we cannot use it in the SERVER side.
ref: https://www.tutorialspoint.com/spring_boot/spring_boot_rest_template.htm